conceptual_model/field-types.md

```diff
@@ -0,0 +1,62 @@
+
+### String field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | string   | any
+| source            | string   | any
+| certificationDate | string   | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+
+
+### Date field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | string   | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+| source            | string   | any
+| certificationDate | string   | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+
+
+### Email field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | string   | [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322#section-3.4)
+| source            | string   | any
+| certificationDate | string   | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+
+
+### Postal address field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | string   | AFNOR NF Z 10-011
+| source            | string   | any
+| certificationDate | dstring  | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+
+
+### Phone number field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | string   | [E.164 Standard](https://www.itu.int/rec/T-REC-E.164/)
+| source            | string   | any
+| certificationDate | string   | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+
+
+### Gender field
+
+| Name              | Type     | Format
+| ------------------|----------|---------------------------------
+| value             | number   | [ISO/IEC 5218](https://en.wikipedia.org/wiki/ISO/IEC_5218)
```

Comment:
For _gender_, FranceConnect gives _male_ or _female_ . Thus, the CMS provider using FranceConnect should convert this value.